src/wp-admin/user-edit.php

```diff
@@ -797,7 +798,7 @@
 			?>
 		</div>
 		<?php else : ?>
```

Comment:
```suggestion
		<?php elseif ( ! wp_is_application_passwords_supported() ) : ?>
```

In manually testing, this is needed to avoid the originally reported issue; else, the HTTP message appears when SSL is enabled but the user doesn't have access or the filter is applied like this:

```php
add_filter( 'wp_is_application_passwords_available', '__return_false' );
```